Ory/kratos/kratos.yml

```
@@ -80,6 +81,13 @@ secrets:
   cipher:
     - 32-LONG-SECRET-NOT-SECURE-AT-ALL
 
+session:
+  cookie:
+    domain: http://openfoodfacts.localhost/
```

See https://www.ory.sh/docs/kratos/guides/configuring-cookies#session-cookies you should not put the `http://` neither a final `/`:
```suggestion
    domain: openfoodfacts.localhost
```

---

fixed